ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_path_groups.schema.yml

```diff
@@ -49,3 +49,20 @@ keys:
               local:
                 type: str
                 description: Optional, if not set, the path-group `name` is used as local.
+        exclude_from_auto_lb_policies:
+          type: bool
+          description: |-
+            When set to `true`, the path-group is not used for auto-generated load-balace policies.
+        default_preference:
```

Comment:
please add an option here callled `excluded` and remove `exclude_from_auto_lb_policies`